bot/exts/evergreen/trivia_quiz.py

```diff
@@ -261,36 +507,46 @@ async def declare_winner(channel: discord.TextChannel, player_data: dict) -> Non
 
                 winners_mention = " ".join(winner.mention for winner in winners)
             else:
-                word = "You"
                 author_index = list(player_data.values()).index(highest_points)
                 winner = list(player_data.keys())[author_index]
                 winners_mention = winner.mention
 
             await channel.send(
                 f"Congratulations {winners_mention} :tada: "
-                f"{word} have won this quiz game with a grand total of {highest_points} points!"
+                f"You have won this quiz game with a grand total of {highest_points} points!"
             )
 
     def category_embed(self) -> discord.Embed:
         """Build an embed showing all available trivia categories."""
-        embed = discord.Embed(colour=discord.Colour.blue())
-        embed.title = "The available question categories are:"
+        embed = discord.Embed(
+            colour=discord.Colour.blue(),
```

Comment:
Same as before, consider using Colours.blue from Lance not d.py

Reply:
done! also, didn't know there was `python_blue` haha